models/repos.js

```diff
@@ -195,6 +195,14 @@ function createManager (multidat, onupdate) {
     var stats = dat.trackStats()
     dat.stats = stats
 
+    multidat.readManifest(dat, function (_, manifest) {
```

Comment:
looks like this fn is only called once, however the metadata should update whenever the `dat.json` changes